controllers/whoToFollow/whoToFollow.js

```diff
@@ -99,20 +108,20 @@ module.exports = async (req, res) => {
         result.push({
           viewtype: 'labeltopic',
           name: topic.name,
-          id: topic.topic_id
+          id: topic.topic_id,
+          users: tempUsers
         });
-
-        result.push(...tempUsers);
       }
     }
+
     for (const indexLocation in LocationsData) {
       const tempUsers = [];
       const location = LocationsData[indexLocation];
 
       for (const indexUserLocation in userLocationFollower) {
         const userLocation = userLocationFollower[indexUserLocation];
         delete userLocation.human_id;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code is deleting the `human_id` field from the `userLocation` object. Similar to the previous comment, if `userLocationFollower` is a reference to an object stored elsewhere in your application, this could unintentionally affect other parts of your application that rely on the `human_id` property. Consider creating a copy of the object before modifying it.
```diff
-         const userLocation = userLocationFollower[indexUserLocation];
-         delete userLocation.human_id;
+         const userLocation = {...userLocationFollower[indexUserLocation]};
+         delete userLocation.human_id;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->